abipy/lessons/lesson_helper_functions.py

```diff
@@ -8,10 +8,34 @@
 import sys
 import os
 import shutil
+import yaml
+import html2text
+from variables import *
 import abipy.abilab as abilab
 import abipy.data as abidata
 
 
+class VariableDatabase(object):
```

Comment:
Are you sure this global variable _VAR_DATABASE is correctly propagated to the other lessons if I import the get_database function in these lessons ? because it didn't work when I tried to save it as a global variable...